pkg/acsengine/transform/apimodel_merger.go

```diff
@@ -8,18 +8,21 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/Azure/acs-engine/pkg/helpers"
+
 	"github.com/Jeffail/gabs"
 	log "github.com/sirupsen/logrus"
 )
 
 // APIModelValue represents a value in the APIModel JSON file
 type APIModelValue struct {
 	stringValue   string
-	intValue      int64
+	intValue      *int64
 	arrayValue    bool
 	arrayIndex    int
```

Comment:
I suppose this could be a pointer as well and we can get rid of the `arrayValue bool`